src/urbit/app/chess.hoon

```diff
@@ -834,49 +834,82 @@
   |=  =path
   ^-  (unit (unit cage))
   ?+  path  (on-peek:default path)
+    ::
+    :: send back a list of chess-games
+      [%x %archive *]
+    ?+  t.t.path  (on-peek:default path)
+    ::
+    ::  =sur -build-file /=chess=/sur/chess/hoon
+    ::  .^((map @dau chess-game:sur) %gx /=chess=/archive/all/noun)
+    ::  .^(json %gx /=chess=/archive/all/json)
+        [%all ~]
+      ``[%chess-archive !>(archive)]
+    ::
+    ::  =sur -build-file /=chess=/sur/chess/hoon
+    ::  .^(chess-archive:sur %gx /=chess=/archive/between/<game id>/<game id>/noun)
+    ::  .^(json %gx /=chess=/archive/between/<game id>/<game id>/json)
+        [%between @ta @ta ~]
+      =/  start=(unit @dau)  `(unit @dau)`(slaw %da i.t.t.path)
+      =/  end=(unit @dau)  `(unit @dau)`(slaw %da i.t.t.path)
+      ``[%chess-archive !>((lot:arch-orm archive start end))]
```

Comment:
relevant for #65 

while this compiles just fine and returns a `mop`, I don't think the `lot:on` is actually excluding any games between the `@ta` ranges I pick.  also, it doesn't seem to be ordered either*. 

_I'll have to come back to it. maybe collapsing the `mop` to a `list` in the %chess-archive mark is causing problems?_

Reply:
See #65; if we use the frontend filter approach, you can handle this from the frontend by walking over the ordered list of metadata and only inserting entries that meet all of the filter criteria into a display set that gets rendered under the archives tab.